### PR TITLE
Create vpc.txt

### PR DESCRIPTION
**Pull request by ayeshabanewale**

### CODE REVIEW

File: vpc.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+resource "aws_vpc" "terraform_test_vpc" {
+  cidr_block           = var.vpc_cidr
+  enable_dns_hostnames = true
+  enable_dns_support   = true
+
+  tags = {
+    Name = var.vpc_tag_name
```
Comment (sanabanewale) [inline vpc.txt:7]: change tag name

**Review by sanabanewale — CHANGES_REQUESTED** [review]
please change

File: vpc.txt (hunk 1 above)
Comment (ayeshabanewale) [reply vpc.txt]: changed please check

Comment (sanabanewale) [reply vpc.txt]: ok

**Pull request merged**